pkg/collector/corechecks/system/cpu_windows.go

```diff
@@ -0,0 +1,160 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+// +build windows
+
+package system
+
+import (
+	"fmt"
+	"strconv"
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/gohai/cpu"
+	log "github.com/cihub/seelog"
+	"golang.org/x/sys/windows"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+)
+
+var (
+	modkernel32 = windows.NewLazyDLL("kernel32.dll")
```

Comment:
Could we move this import to a its own file since it's user by multiple check now. Like `utils_windows.go` or something like that.